test/unit/core.js

```diff
@@ -1190,6 +1190,18 @@ test("jQuery.each(Object,Function)", function() {
 	equal( i, document.styleSheets.length, "Iteration over document.styleSheets" );
 });
 
+test("jQuery.each/map(undefined/null,Function)", 1, function() {
```

Comment:
Oh it's because I am testing :
``` javascript
$.each( null || undefined, function() {
});
```

Like the title of the precedent test ( *jQuery.each(Object,Function)* )